protocols/imap/src/test/java/org/apache/james/imap/processor/base/FakeMailboxListenerFlagsUpdate.java

```diff
@@ -31,27 +31,11 @@
 
 public class FakeMailboxListenerFlagsUpdate extends FlagsUpdated {
```

Comment:
why do we need this class in the first place ?

Reply:
I removed this class